packages/contentful--react-apps-toolkit/package.json

```
@@ -31,7 +31,8 @@
     "pre-commit": "lint-staged"
   },
   "peerDependencies": {
-    "react": ">=16.8.0"
+    "react": ">=16.8.0",
+    "@contentful/app-sdk": "^4.0.0"
```

we should allow all versions above

```suggestion
    "@contentful/app-sdk": ">=4.0.0"
```